src/bundle/Resources/public/scss/_filters.scss

```diff
@@ -46,6 +48,29 @@
         }
     }
 
+    &:before,
+    &:after {
+        content: '';
+        position: absolute;
+        border-style: solid;
+        border-width: 0 10px 10px;
```

Comment:
`calculateRem(10px)` and similarly in other places where `px` unit is used. Let's be consistent :)

Reply:
I meant `border-width: 0 calculateRem(10px) calculateRem(10px);`